### DISCUSSION TITLE
在线编译（普通用户使用），请问针对windows，怎么设置Icon？

### DISCUSSION THREAD

**Discussion by xukai504087493** [tw93/Pake#775] [Q&A]
无法上传自己的图片作为icon，这里我尝试了在线图床，也不行。针对windows，我看了window的编译，是不是要上传三张图片？但是在线编译这里只有一个参数，请问如果要在线编译windows的，怎么设置这个Icon参数？
<img width="315" alt="image" src="https://github.com/user-attachments/assets/75a0989b-6794-4283-b381-00ec8c6b2c8b">

参考“在线编译（普通用户使用）”：
https://github.com/tw93/Pake/wiki/%E5%9C%A8%E7%BA%BF%E7%BC%96%E8%AF%91%EF%BC%88%E6%99%AE%E9%80%9A%E7%94%A8%E6%88%B7%E4%BD%BF%E7%94%A8%EF%BC%89

**Comment by tw93** [comment]
icon 可以传递到oss cdn，或者github上当做一个资源的方式，只要可以接口请求到就好了

**Reply by xukai504087493** [reply]
感谢，icon测试过mac 的，因为mac只有一个文件，直接设置图床地址是可以正确生成的。
但是windows的，需要设置三个图片地址（如下图说明），但是上面图示里的“[Icon,Optional]”参数地址只能传递一个，这个问题应该怎么解决呢？
<img width="822" alt="image" src="https://github.com/user-attachments/assets/d160a8e7-4926-44af-9233-7f9d49c0e46f">

**Comment by Lurito** [comment]
Windows多分辨率ico可以使用GIMP软件，对不同分辨率进行图层堆叠来生产。可以安装GIMP后打开一些软件的ICO作为参考。
